### DISCUSSION TITLE
A few more simple text-based extensions

### DISCUSSION THREAD

**Discussion by Flappy25** [TurboWarp/extensions#635] [Show and tell]
A collection of extensions to use, if you want to use them then a simple credit comment in the backdrop sprite is good enough

<b>Base64 Converter:</b>
[base64 encryption.txt](https://github.com/TurboWarp/extensions/files/11856490/base64.encryption.txt)

<b>Clipboard:</b>
[clipboard.txt](https://github.com/TurboWarp/extensions/files/11856492/clipboard.txt)

<b>Lorem Ipsum Generator:</b>
[lorem ipsum gen.txt](https://github.com/TurboWarp/extensions/files/11856496/lorem.ipsum.gen.txt)

<b>Morse Code:</b>
[morse code.txt](https://github.com/TurboWarp/extensions/files/11856497/morse.code.txt)

<b>Javascript Random Numbers Generator:</b>
[simple random numbers.txt](https://github.com/TurboWarp/extensions/files/11856498/simple.random.numbers.txt)

<b>Text Analyzer:</b>
[text analyzer.txt](https://github.com/TurboWarp/extensions/files/11856499/text.analyser.txt)

<b>Text Comparison:</b>
[Text Comparison.txt](https://github.com/TurboWarp/extensions/files/11856500/Text.Comparison.txt)

<b>Text Encryption and Decryption:</b>
[Text Encryption and Decryption.txt](https://github.com/TurboWarp/extensions/files/11856501/Text.Encryption.and.Decryption.txt)

I will add more soon

**Comment by CST1229** [comment]
> Base64 Encryption

this hurts to read (no offense btw)

**Reply by Flappy25** [reply]
> > Base64 Encryption
> 
> this hurts to read (no offense btw)

Why? I seems fine to me

**Reply by CST1229** [reply]
Base64 is not encryption, it's just encoding. While it can't be read by humans, it can very easily be converted back into text.

**Reply by Flappy25** [reply]
> Base64 is not encryption, it's just encoding. While it can't be read by humans, it can very easily be converted back into text.

Ok sorry

**Comment by ghost** [comment]
Nice! But unfortunately, clipboard is broken...

**Comment by Flappy25** [comment]
> Nice! But unfortunately, clipboard is broken...

Load it unsandboxed
